spec/src/main/asciidoc/configuration.asciidoc

```diff
@@ -257,6 +257,25 @@ Reloading the Public Key from the location at runtime as well as the frequency o
 reloading is beyond the scope of this specification and any such feature should be
 considered vendor-specific.
 
+#### `mp.jwt.token.header`
+
+The `mp.jwt.token.header` config property allows to set up the header which is expected to contain a JWT token.
+
+When supplied, `mp.jwt.token.header` will override the standard `Authorization` header to retrieve the JWT token.
+
+#### `mp.jwt.token.scheme`
+
+The `mp.jwt.token.scheme` config property allows to set up the HTTP Authentication Scheme which is expected to use with
+the Authorization header.
+
+When supplied, `mp.jwt.token.scheme` will override the standard `Bearer` scheme to retrieve the JWT token.
+
+#### `mp.jwt.token.cookie`
+
+The `mp.jwt.token.cookie` config property allows to set up the Cookie name which is expected to contain a JWT token.
+
+This config property must be used together with `mp.jwt.token.header` set to `Cookie` or else it will be ignored.
```

Comment:
@radcortez Perhaps `This property will be ignored unless 'mp.jwt.token.header' is set to 'Cookie'` ? Please also expand `config` -> `configuration` elsewhere.

Reply:
Ok!